components/infobox/commons/infobox_manufacturer.lua

```diff
@@ -105,7 +105,6 @@ function Manufacturer:setLpdbData(args)
 		image = args.image,
 		imagedark = args.imagedark,
 		extradata = {
-			region = args.region,
 			status = args.status,
 			locations = Locale.formatLocations(args),
```

Comment:
The locations? I won't lie I can't remember exactly why it was there, I think it was so it could be used in API calls for flags maybe? Could be removed and reintroduced when there's an actual need for it